docs/setup-guide/setup-dev-env.md

```diff
@@ -1,28 +1,21 @@
-## Set up developer environment
+## Set up developer environment (updated on 2021-07-22)
 
-Note: tested on AWS EC2 Ubuntu 16.04 x86 image.
- 
+Note: Tested on AWS EC2 Ubuntu 16.04/18.04/20.04 x86 image.
 
 ### Clone repo
 ```
 $ mkdir -p go/src/
 $ cd go/src/
-$ git clone https://github.com/futurewei-cloud/arktos
+$ git clone https://github.com/centaurusinfra/arktos
 ```
 
-Note: the following steps can be simplified by running hack/setup-dev-node.sh
-
-### Install Golang
+### Install needed packages (docker, make, gcc, jq and golang)
 ```
-$ sudo apt-get update
-# $ sudo apt-get -y upgrade // optional
-$ cd /tmp
-$ wget https://dl.google.com/go/go1.12.9.linux-amd64.tar.gz
-$ tar -xvf go1.12.9.linux-amd64.tar.gz
-$ sudo mv go /usr/local
-$ rm go1.12.9.linux-amd64.tar.gz
+$ ./hack/setup-dev-node.sh
```

Comment:
Got error after run "./hack/setup-dev-node.sh":
```
Please add the following lines into your shell profile ~/.profile.
    GOROOT=/usr/local/go
    GOPATH=/home/sonyafenge/go
hack/setup-dev-node.sh: line 49: GOPATH: unbound variable
```

Reply:
Fixed the issue. Thanks!